P5/Exemplars/tei_rdfa.tei

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="tei_rdfa.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="tei_rdfa.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">   
+   <teiHeader>
+      <fileDesc>
+         <titleStmt about="disco:au_004e">
+            <title property="dc:title"><!-- supply a title --></title>
+            <author property="dc:creator"><!-- supply an author --></author>            
+         </titleStmt>
+         <publicationStmt>
+            <p><!-- supply publication information--> </p>
+         </publicationStmt>
+         <sourceDesc>
+            <p><!-- supply information about the source --></p>
+         </sourceDesc>
+      </fileDesc>
+      <profileDesc>
+         <particDesc>
+            <listPerson>
+               <person xml:id="disco_004e" about="disco:004e" typeof="foaf:Person">
```

Comment:
I could make an argument that the `@about` attribute is redundant here....maybe something to think about.